docs/features/offloading.md

```diff
@@ -1,10 +1,27 @@
 # Offloading
 
-**Workload offloading** is enabled by the **virtual node** abstraction.
-A virtual node is spawned at the end of the peering process in the local (i.e., consumer) cluster, and represents (and aggregates) the subset of resources shared by the remote cluster.
+**Workload offloading** is enabled by a **virtual node**, which is spawned in the local (i.e., consumer) cluster at the end of the peering process, and represents (and aggregates) the subset of resources shared by the remote cluster.
+
 This solution enables the **transparent extension** of the local cluster, with the new node (and its capabilities) seamlessly taken into account by the vanilla Kubernetes scheduler when selecting the best place for the workloads execution.
 At the same time, this approach is fully compliant with the **standard Kubernetes APIs**, hence allowing to interact with and inspect offloaded pods just as if they were executed locally.
 
+## Assigned resources
+
+By default, the virtual node is assigned with 90% of the resources available in the remote cluster. For example:
+
+* If the remote cluster has 100 vCPUs available, the virtual node created with 90 vCPUs.
+* If now the remote cluster starts some applications that consume 50 vCPUs (i.e., pods _requesting_ resources), the virtual node is resized to 45 vCPUs (i.e., 90% of (100-50)).
+* If the remote cluster has some autoscaling mechanism that, at some point, double the size of the cluster, which reaches 200 vCPUs (all of them unused by any pod), the virtual node will be resized with 180 vCPUs.
+
+This mechanism applies to all the physical resources available in the remote cluster, e.g., CPUs, RAM, GPUs and more.
+The percentage of sharing can be customized also at run-time using the `--sharing-percentage` option, as documented in the proper [section](https://docs.liqo.io/en/latest/installation/install.html#control-plane) of the Liqo installation.
+
+**Warning 1**: pay attention to _math rounding_. For instance, if your remote cluster has 1 GPU, with default settings the virtual node will be set with 0.9 GPUs. Since numbers must be integers, you may end up with a virtual node with _zero_ GPUs.
```

Comment:
```suggestion
```{warning}
Pay attention to _math rounding_. For instance, if your remote cluster has 1 GPU, with default settings the virtual node will be set with 0.9 GPUs. Since numbers must be integers, you may end up with a virtual node with _zero_ GPUs.
```
```

It is just an aetetich suggestion.

Reply:
![image](https://user-images.githubusercontent.com/33266330/230153405-3c159264-b579-46b7-84b6-9eb76f092bd3.png)